schemas/v3.1/schema.yaml

```
@@ -0,0 +1,809 @@
+        patternProperties:
+          ^/:
+            type: object
+            oneOf:
+              - $ref: '#Reference'
+              - $ref: '#PathItem'
+        unevaluatedProperties: false
+      PathItem:
+        $anchor: PathItem
+        type: object
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/Describable'
+          - $ref: '#/$defs/Mixins/$defs/Extensible'
+        properties:
+          summary:
+            type: string
+          servers:
+            type: array
+            items:
+              $ref: '#Server'
+          parameters:
+            type: array
+            items:
+              oneOf:
+                - $ref: '#Parameter'
+                - $ref: '#Reference'
+            uniqueItems: true
+      Operation:
+        $anchor: Operation
+        type: object
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/Describable'
+          - $ref: '#/$defs/Mixins/$defs/Deprecatable'
+          - $ref: '#/$defs/Mixins/$defs/Extensible'
+        properties:
+          tags:
+            type: array
+            items:
+              type: string
+          summary:
+            type: string
+          externalDocs:
+            $ref: '#ExternalDocumentation'
+          operationId:
+            type: string
+          parameters:
+            type: array
+            items:
+              $ref: '#Parameter'
+          requestBody:
+            $ref: '#RequestBody'
+          responses:
+            $ref: '#Responses'
+          callback:
+            $ref: '#Callback'
+          security:
+            type: array
+            items:
+              $ref: '#SecurityRequirement'
+          servers:
+            type: array
+            items:
+              $ref: '#Server'
+        unevaluatedProperties: false
+      Responses:
+        $anchor: Responses
+        type: object
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/Extensible'
+        properties:
+          default:
+            $ref: '#Response'
+        patternProperties:
+          '[1-5](?:\d{2}|XX)':
+            $ref: '#Response'
+        minProperties: 1
+        unevaluatedProperties: false
+      SecurityRequirement:
+        $anchor: SecurityRequirement
+        type: object
+        additionalProperties:
+          type: array
+          items:
+            type: string
+      Tag:
+        $anchor: Tag
+        type: object
+        required:
+          - name
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/Describable'
+          - $ref: '#/$defs/Mixins/$defs/Extensible'
+        properties:
+          name:
+            type: string
+          externalDocs:
+            $ref: '#ExternalDocumentation'
+        unevaluatedProperties: false
+      ExternalDocumentation:
+        $anchor: ExternalDocumentation
+        allOf:
+          - $ref: '#/$defs/Mixins/$defs/Extensible'
+        type: object
+        required:
+          - url
+        properties:
+          description:
+            type: string
```

Consider replacing the `description` property with `#/$defs/Mixins/$defs/Describable` for consistency with other definitions.
```suggestion
        allOf:
          - $ref: '#/$defs/Mixins/$defs/Extensible'
          - $ref: '#/$defs/Mixins/$defs/Describable'
        type: object
        required:
          - url
        properties:
```